app/views/organizations/show.html.erb

```diff
@@ -1,18 +1,25 @@
-<h1><%= @organization.name %></h1>
+<div class="position-relative overflow-hidden mb-3 bg-light">
```

Comment:
I'm good with this. Just pinning this idea for future, but maybe we want to make this a content_for as well as so other pages can easily tap into this design element also